src/pages/dev/amplifier/chain-integration/integrate-a-chain.mdx

```diff
@@ -11,6 +11,77 @@ The Axelar Virtual Machine (AVM) and Amplifier are currently under active develo
+    ```
+    
+    Leave this process running in the background, and perform additional commands in a new terminal window or shell.
+   
+1. Download [`ampd`](https://github.com/axelarnetwork/axelar-amplifier/tree/main/ampd#worker-onboarding), the off-chain daemon responsible for voting and signing with the amplifier protocol. 
+
+    We recommend downloading the `ampd` [binary](https://github.com/axelarnetwork/axelar-amplifier/releases/tag/ampd-v0.1.0) ([Linux](https://github.com/axelarnetwork/axelar-amplifier/releases/download/ampd-v0.1.0/ampd-linux-amd64-v0.1.0), [Apple Silicon Mac](https://github.com/axelarnetwork/axelar-amplifier/releases/download/ampd-v0.1.0/ampd-darwin-arm64-v0.1.0), [Intel Mac](https://github.com/axelarnetwork/axelar-amplifier/releases/download/ampd-v0.1.0/ampd-darwin-amd64-v0.1.0)). All instructions in this tutorial will assume you have a binary in your `PATH` named `ampd`.
+
+        - You can add the `ampd` binary to your path with `sudo ln -s <BINARY NAME> /usr/local/bin/ampd`.
+        - Alternatively, you can set an alias with `alias ampd=<BINARY PATH>` and restart your terminal. 
+        
+    You should make the binary executable with `chmod a+x ./ampd-darwin-arm64-v0.1.0`, customizing the binary name to match the one you downloaded.
+
+    If you don't want to add it to your path, you can refer to the binary directly in every command in this tutorial, such as `~/ampd-linux-amd64-v0.1.0 <command>`.
+
+    If you get a security warning while trying to run `ampd` on Mac, you can disable the [gatekeeper](https://support.apple.com/guide/security/gatekeeper-and-runtime-protection-sec5599b66df/web) for a single binary with `sudo xattr -d com.apple.quarantine ./ampd-darwin-arm64-v0.1.0` or disable gatekeeper globally with `sudo spctl --master-disable`.
+
+    There are two other ways to install `ampd`. 
+        - You can use [`ampd` from Docker](https://hub.docker.com/r/axelarnet/axelar-ampd). Make sure you pull the `0.1.0` version and `amd64` platform specifically, i.e. `docker pull axelarnet/axelar-ampd:v0.1.0 --platform amd64`. 
+        - You can install `ampd` from source by cloning and building the [Axelar Amplifier repository](https://github.com/axelarnetwork/axelar-amplifier).
+
+`ampd` needs access to a locally running `tofnd` instance to onboard as a verifier or run the daemon. See the [tofnd repository](https://github.com/axelarnetwork/tofnd) for more info.
+
+1. Create an ampd configuration file at `~/.ampd/config.toml` (or choose a different location and refer to your chosen location later in ampd commands such as `ampd -config /home/me/devnet-amplifier.toml`). This configuration file tells `ampd` how to connect to your local `tofnd`, how to talk to the devnet and the Amplifier smart contracts, and includes a default configuration for verifying transactions on the Avalanche testnet.
+
+    The following is an example `config.toml` file for the Amplifier devnet:
+
+    ```bash
```

Comment:
Do you think we should have these files inline or downloadable as separate files?

this isn't bash code, but I know our code colorer (prism) isn't configured for toml, should we add it?

Reply:
I think inline code is good in this case, because the dev can see the config before using it themselves. I'm also ok with a GitHub link.

Configuring our code colorer to format toml would be great. This formatting was copied from the verifier onboarding docs, so it's configured to bash over there, too.